src/main/resources/assets/terrestria/lang/en_gb.json

```diff
@@ -0,0 +1,282 @@
+{
```

Comment:
Only include the stuff you actually changed in this language file, it will fallback to en_us